src/App.jsx

```diff
@@ -1,35 +1,14 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+// eslint-disable-next-line no-unused-vars
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Greeting from './components/Greeting';
```

Comment:
- [ ] It's not a good practise to disable linters, kindly resolve the issue that the linter is pointing out. Thank you. 🙏